CSharpMath/Atom/LaTeXSettings.cs

```diff
@@ -417,7 +421,7 @@ public static class LaTeXSettings {
       if (atomWithoutScripts is IMathListContainer container)
         foreach (var list in container.InnerLists)
           list.Clear();
-      return CommandSymbols.TryGetBySecond(atomWithoutScripts, out var name) ? name : null;
+      return CommandSymbols.SecondToFirst.TryGetValue(atomWithoutScripts, out var name) ? name : null;
```

Comment:
The `FirstToSecond.TryGetValue` and `SecondToFirst.TryGetValue` are repetitive. Why can't we keep `TryGetByFirst` and `TryGetBySecond`?

Reply:
Because you'd need to add all these:
```
public bool TryGetByFirst(TFirst first, out TSecond second) =>
  firstToSecond.TryGetValue(first, out second);
/// <summary> Gets the primary alias corresponding to second.</summary>
public bool TryGetBySecond(TSecond second, out TFirst first) =>
  secondToFirst.TryGetValue(second, out first);
public bool ContainsByFirst(TFirst first) => firstToSecond.ContainsKey(first);
public bool ContainsBySecond(TSecond second) => secondToFirst.ContainsKey(second);
public TSecond GetByFirst(TFirst first) => firstToSecond[first];
/// <summary> Gets the primary alias corresponding to second.</summary>
public TFirst GetBySecond(TSecond second) => secondToFirst[second];
```
And then add two Counts and then remove the dictionaries. It's a lot less clean to manually expose all the dictionary properties like this. The code needed to do it is much than the code needed above.

Reply:
I see. @FoggyFinder?

Reply:
Sorry, I didn't notice the ping. Yeah, fine. All these methods can be re-added in the future as needed.

Reply:
Alright, this is resolved.